### PR TITLE
Wrapper functions for function_symbols, get_name and coeff

### PR DESCRIPTION
**Pull request by ahumenberger**

### CODE REVIEW

**Comment by isuruf** [comment]
Looks good to me. Can you add a test?

**Comment by ahumenberger** [comment]
Note that the test for `coeff(expr, x, Basic(0))` fails right now, as it depends on a fix in SymEngine itself.

File: test/runtests.jl [hunk 1]
```diff
@@ -217,3 +217,19 @@ f = SymFunction("f")
...
+@test get_args(f(n+1)) == [n+1]
+@test get_args(f(n, m^2)) == [n, m^2]
+
+# Coefficients
+@vars x y
+expr = x^3 + 3*x^2*y + 3*x*y^2 + y^3 + 1
+@test coeff(expr, x, Basic(3)) == 1
+@test coeff(expr, x, Basic(2)) == 3*y
+@test coeff(expr, x, Basic(1)) == 3*y^2
+@test coeff(expr, x, Basic(0)) == y^3 + 1
```
Comment (isuruf) [inline test/runtests.jl]: Can you do this test only for,
```
SymEngine.libversion >= VersionNumber("0.4.0")
```

**Comment by isuruf** [comment]
Thanks

**Pull request merged**